tests/unit/test_utils.py

```diff
@@ -369,3 +369,36 @@ def test_rmtree_retries_for_3sec(tmpdir, monkeypatch):
     monkeypatch.setattr(shutil, 'rmtree', Failer(duration=5).call)
     with pytest.raises(OSError):
         rmtree('foo')
+
+class Test_normalize_path(object):
+    def setup(self):
+        self.tempdir = tempfile.mkdtemp()
+        self.orig_working_dir = os.getcwd()
+        os.chdir(self.tempdir)
+
+    def teardown(self):
+        os.chdir(self.orig_working_dir)
+        shutil.rmtree(self.tempdir, ignore_errors=True)
+
+    # Technically, symlinks are possible on Windows, but you need a special
+    # permission bit to create them, and Python 2 doesn't support it anyway, so
+    # it's easiest just to skip this test on Windows altogether.
+    @pytest.mark.skipif("sys.platform == 'win32'")
+    def test_resolve_symlinks(self):
+        d = os.path.join('foo', 'bar')
+        f = os.path.join(d, 'file1')
+        os.makedirs(d)
+        with open(f, 'w'): pass  # Create the file
+
+        os.symlink(d, 'dir_link')
+        os.symlink(f, 'file_link')
+
+        assert normalize_path('dir_link/file1', resolve_symlinks=True) \
+                == os.path.join(self.tempdir, f)
```

Comment:
The Travis pep8 tests are failing on this line (and the similar lines below), with:

```
E127 continuation line over-indented for visual indent
```

I can't find an indentation that makes it happy - one extra space of indentation is still 'over-indented', and zero spaces gives another warning about 'missing indentation'. What is the desired formatting for a statement like this?